Civi.php

```diff
@@ -75,7 +75,7 @@ public static function lockManager() {
   }
 
   /**
-   * @return \Psr\Log\LoggerInterface
+   * @return \CRM_Core_Error_Log
```

Comment:
I feel like this *could*not  be something else in future so Logger might be better

Reply:
Well currently it always returns CRM_Core_Error_Log but if that changes we can update the annotation.